nir/src/main/scala/scala/scalanative/nir/Arg.scala

```diff
@@ -0,0 +1,13 @@
+package scala.scalanative.nir
+
+case class Arg(ty: Type, passConvention: Option[PassConvention] = None)
+
+sealed abstract class PassConvention
```

Comment:
`PassConv` is clear enough and lets one have reasonably short default type name without having to use renaming imports.